src/main/java/org/jenkinsci/plugins/workflow/graphanalysis/ForkScanner.java

```diff
@@ -235,7 +235,6 @@ interface FlowPiece {  // Mostly a marker
     // TODO see if this can be replaced with a FlowChunk acting as a container class for a list of FlowNodes
     static class FlowSegment implements FlowPiece {
         ArrayList<FlowNode> visited = new ArrayList<>();
-        FlowPiece after;
```

Comment:
Was not ever used apart from test assertions and my simplifications left it always `null`, making it completely pointless.